src/sage/groups/perm_gps/permgroup.py

```diff
@@ -4106,6 +4106,33 @@ def isomorphism_type_info_simple_group(self):
         else:
             raise TypeError("group must be simple")
 
+    def minimal_normal_subgroups(self):
+        """
+        Return a list containing those nontrivial normal subgroups of the group that are minimal among the nontrivial normal subgroups.
+
+        EXAMPLES::
+
+            sage: G = PermutationGroup([(1,2,3),(4,5)])
+            sage: G.minimal_normal_subgroups()
+            [Subgroup generated by [(4,5)] of (Permutation Group with generators [(4,5), (1,2,3)]),
+             Subgroup generated by [(1,2,3)] of (Permutation Group with generators [(4,5), (1,2,3)])]
+        """
+        return [self.subgroup(gap_group=gap_subgroup) for gap_subgroup in self.gap().MinimalNormalSubgroups()]
```

Comment:
```suggestion
        return [self.subgroup(gap_group=gap_subgroup)
                for gap_subgroup in self.libgap().MinimalNormalSubgroups()]
```
We want to move things to libgap (direct library interface), and if it doesn't use libgap, this redirects to a gap call.